src/analyses/reaching_definitions.cpp

```diff
@@ -19,6 +20,16 @@ Date: February 2013
 
 #include "reaching_definitions.h"
 
+reaching_definitions_analysist::reaching_definitions_analysist(
+  const namespacet &_ns):
+    concurrency_aware_ait<rd_range_domaint>(),
+    ns(_ns)
+{
+}
+
+
+reaching_definitions_analysist::~reaching_definitions_analysist()=default;
```

Comment:
Just a single blank line in between, and function comment headers for both of these.